articles/api-management/api-management-using-with-internal-vnet.md

```diff
@@ -96,6 +96,9 @@ Learn how to [set up a private zone in Azure DNS](../dns/private-dns-getstarted-
 > * Direct management endpoint
 > * Git
 
+>	Specific reserved zone names (azure-api.net) will be blocked from creation to prevent disruption of services in Private DNS zone as mentioned [here](../dns/private-dns-privatednszone.md).
+>	So for default APIM Gateway endpoint - APIMserviceName.azure-api.net, create A-record in Custom DNS server associated to APIM Vnet.
```

Comment:
```suggestion
>	So, for the default API endpoint ({APIMserviceName}.azure-api.net), create an A-record in the custom DNS server associated to the API Management VNet.
```

Reply:
Minor editorial suggestion above

Reply:
Thanks for sharing your feedback.
There is a small edit that has to be done in the updated change.
So, for the default **API Management Gateway endpoint** ({APIMserviceName}.azure-api.net), create an A-record in the custom DNS server associated to the API Management VNet.